pegasus/sites.v3/code.org/public/css/hoc-skinny-banner.scss

```diff
@@ -60,5 +60,10 @@ aside.hoc-skinny-banner {
     border-color: $neutral_white;
     color: $neutral_dark;
     font-family: $gotham-bold;
+
+    &:hover {
+      background-color: $neutral_dark10;
+      border-color: $neutral_dark10;
+    }
   }
```

Comment:
Can you change the `h1` font-size on this banner to `2.15em`? ([link to line in file](https://github.com/code-dot-org/code-dot-org/blob/9c127dfa931fa60cff36f31bc06ece41e46b8fa4/pegasus/sites.v3/code.org/public/css/hoc-skinny-banner.scss#L30)) 

This change and changing the line-height to 1.2 ([ref](https://github.com/code-dot-org/code-dot-org/pull/49831/files#r1073768191)) should fix the small h1 text so it looks like this:
<img width="991" alt="Screenshot 2023-01-18 at 8 30 08 AM" src="https://user-images.githubusercontent.com/9256643/213237613-637e68eb-f183-4ffe-942d-9958caf9d9c8.png">